shared/macros/10-bash.jinja

```diff
@@ -1598,6 +1598,18 @@ fi
 :type format: str
 
 #}}
+
+{{%- macro bash_comment_config_line(config_file, key) -%}}
```

Comment:
This macro needs to be tested by a BATS test.

Reply:
Very useful indeed, haven't had the BATS pleasure so far :+1: done in 4438714d16e0939f61d3759949b672b03fa5832f